### DISCUSSION TITLE
How do you tell / debug if a custom script is running

### DISCUSSION THREAD

**Discussion by go4ino** [violentmonkey/violentmonkey#1879] [Q&A] [answered]
Hello I'm trying to get the [Tumblr unfucker](https://github.com/enchanted-sword/dashboard-unfucker) script to work and it doesn't seem to run at all.

How to I debug and check currently running violentmonkey scripts? Checking firefox `f12` console / debugger menus doesn't display anything running with the userscript name I'm trying to verify is running, which makes me think the script isn't running at all, but I'm not sure how to confirm that.

**Comment by tophf** [comment, marked as answer]
1. Click the Violentmonkey icon in the toolbar to open the popup that lists all running scripts for the page
2. In devtools you should first go to its settings and tick `Enable browser chrome and add-on debugging` option, then close and re-open devtools, then you'll see the script in the Debugger panel: ![image](https://github.com/violentmonkey/violentmonkey/assets/1310400/443255b5-da12-400a-ab21-1fcbaf84ecaa)

**Comment by go4ino** [comment]
Thank for the reply.

For some reason enabling this option got the script to start running
